Samples/auth/Outlook-Add-in-SSO/README.md

```diff
@@ -18,88 +18,37 @@ description: "An Outlook add-in sample that accesses Microsoft Graph using singl
 
 - [Use the SSO access token](https://learn.microsoft.com/outlook/add-ins/authenticate-a-user-with-an-sso-token) to call the Microsoft Graph API without prompting the user
 - If the SSO token is not available, authenticate to the user's OneDrive using the [Microsoft Authentication Library (MSAL)](https://learn.microsoft.com/azure/active-directory/develop/msal-overview).
 - Use the [Microsoft Graph API](https://developer.microsoft.com/graph/docs/api-reference/v1.0/resources/onedrive) to create files in OneDrive
-- Add [add-in command buttons](https://learn.microsoft.com/outlook/add-ins/add-in-commands-for-outlook) to the Outlook ribbon when reading mail, including a UI-less button and a button that opens a task pane
 - Implement a WebAPI that uses the [Microsoft identity platform and OAuth 2.0 On-Behalf-Of flow](https://learn.microsoft.com/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow) to exchange the user's access token for a new access token with permissions to Microsoft Graph and OneDrive.
 
-## Register the add-in with Azure AD v2.0 endpoint
-
-1. Navigate to the [Azure portal - App registrations](https://go.microsoft.com/fwlink/?linkid=2083908) page to register your app.
-
-1. Sign in with the ***admin*** credentials to your Microsoft 365 tenancy. For example, MyName@contoso.onmicrosoft.com.
-
-1. Select **New registration**. On the **Register an application** page, set the values as follows:
-
-    * Set **Name** to `AttachmentsDemo`.
-    * Set **Supported account types** to **Accounts in any organizational directory (Any Azure AD directory - Multitenant) and personal Microsoft accounts (e.g. Skype, Xbox)**.
-    * In the **Redirect URI** section, ensure that **Web** is selected in the dropdown and then set the URI to `https://localhost:44355/AzureADAuth/Authorize`.
-    
-    **Note:** The port number used for the redirect URI (`44355`) must match the port your web server is running on. When you open the Visual Studio solution in later steps, you can find the web server's port number by selecting the **AttachmentDemoWeb** project in **Solution Explorer**, then looking at the **SSL URL** setting in the properties window.
-
-1. Choose **Register**.
-
-1. On the **AttachmentsDemo** page, copy and save the **Application (client) ID**. You'll use it in later procedures.
+## Register the add-in with Microsoft identity platform
 
-1. Under **Manage**, select **Certificates & secrets**. Select the **New client secret** button. Enter a value for **Description**, then select an appropriate option for **Expires**, and choose **Add**.
+Use the following value for the placeholder for the subsequent app registration steps.
 
-1. Copy and save the client secret value. You'll use it in later procedures.
+| Placeholder or section | Value                               |
+|---------------------|-------------------------------------|
+| <add-in-name>       | AttachmentDemo                      |
```

Comment:
The angle brackets are causing this to not render correctly.

![image](https://user-images.githubusercontent.com/38896772/205398280-22f21f82-64cd-4f79-8137-7a65e97f3672.png)